content/d3/concepts/scaling/terms/range/range.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.range()'
+Description: 'Used to define output range of a scale.'
+Subjects:
+  - 'Data Visualization'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Data'
+  - 'Graphs'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/computer-science'
+---
+
+The **`.range()`** method is used to define the output range of a scale. The range represents the output values produced by the scale when mapped from the `domain`. The syntax for the `.range()` method depends on the type of scale used.
+
+## Syntax
+
+```pseudo
+d3.scaleType()
+  .domain([minValue, maxValue])
+  .range([minOutput, maxOutput]);
+```
+
+- **.scaleType():** The selected scale method (e.g., `.scaleLinear()`- a continuous linear scale).
+- ••.domain():** Requires the `minValue` and `maxValue` arguments to set the accepted range.
+- **.range():** The second step in defining a scale, the arguments `minOutput` and `maxOutput` are used to specify the set of values returned by the scale.
```

Comment:
```suggestion
- `.scaleType()`: The selected scale method (e.g., `.scaleLinear()`- a continuous linear scale).
- `.domain()`: Requires the `minValue` and `maxValue` arguments to set the accepted range.
- `.range()`: The second step in defining a scale, the arguments `minOutput` and `maxOutput` are used to specify the set of values returned by the scale.
```
Use backticks to indicate methods or reserved keywords. Change the last line as it contains plagiarism (Line of `.range()`)